examples/client_example/client_example.py

```diff
@@ -73,8 +73,8 @@ def download(target: str) -> bool:
         path = updater.download_target(info)
         print(f"Target downloaded and available in {path}")
 
-    except (OSError, RepositoryError) as e:
-        print(str(e))
+    except (OSError, RepositoryError, DownloadError) as e:
```

Comment:
Do I understand correctly that any download related errors were just not caught here before, and that re-raising all of them as DownloadErrors allows us to easily handle them too here?

Reply:
they were handled, but only because requests errors all happen to be derived from OSError -- so this was relying on a fetcher implementation detail.